tests/integration/test_arrowflight_interface/configs/users.xml

```diff
@@ -0,0 +1,11 @@
+<clickhouse>
+    <users>
+        <default>
+            <password></password>
+            <profile>default</profile>
+            <quota>default</quota>
```

Comment:
It seems only `<password>` is necessary here.

Reply:
fixed